### ISSUE TITLE
Code Style / formatter : preview does not work for Align Properties

### ISSUE THREAD

**Issue by rakk** [VladRassokhin/intellij-hcl#288]
### Prerequisites
* running version 0.7.7
* searched for any *Align* and *Formatter*

### Installation details
 - IDE version
```
IntelliJ IDEA 2019.3.1 (Community Edition)
Build #IC-193.5662.53, built on December 18, 2019
Runtime version: 11.0.5+10-b520.17 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 2000M
Cores: 4
Registry: 
Non-Bundled Plugins: com.dmarcotte.handlebars, org.intellij.plugins.hcl
```
* intellij-hcl plugin `version: 0.7.7`
* Terraform version: `v0.12.19`

### Terraform Configuration Files
N/A

### Exception
N/A

### Expected Behavior
When I open **IntelliJ Settings** -> **Editor** -> **Code Style** -> **Terraform Config**
Then open tab: **Other**
Then  select **On equals**
Then the preview (next to right) is updated accordingly: 
```hcl
simple           = true
pa.int           = false
under_scr        = 1
mi-nus            = 'yep'

#another logical block
_5                    = true
w1th.num8er5 = 'acceptable'
```

### Actual Behavior

Preview does not change at all.

```hcl
simple = true
pa.int = false
under_scr = 1
mi-nus = 'yep'

#another logical block
_5 = true
w1th.num8er5 = 'acceptable'
```

This suggests that the feature is not working. But actually is working fine.

### Steps to Reproduce

1. When I open **IntelliJ Settings** -> **Editor** -> **Code Style** -> **Terraform Config**
2. Then open tab: **Other**
3. Then  select **On equals**

**Comment by alFReD-NSH** [comment]
I can replicate this.

**Comment by nmarchini** [comment]
I have this issue also, any news on when a fix can be put in place?

**Comment by VladRassokhin** [comment]
Fixed in 0.7.14
